airflow/config_templates/config.yml

```diff
@@ -720,6 +720,14 @@ database:
       type: string
       example: ~
       default: "True"
+    external_db_managers:
+      description: |
+        List of DB managers to use to migrate external tables in airflow database. The managers must inherit
+        from BaseDBManager
+      version_added: 3.0.0
+      type: string
+      example: ~
+      default: "airflow.providers.fab.auth_manager.models.db.FABDBManager"
```

Comment:
The default should be null I think. We don't want things to blow up if you don't have the FAB provider installed.

Reply:
Hmm, yeah, good call.